WalletWasabi.Fluent.Generators/WalletWasabi.Fluent.Generators.csproj

```diff
@@ -10,7 +10,6 @@
 		<RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
 		<DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
 		<InvariantGlobalization>true</InvariantGlobalization>
-		<RuntimeIdentifiers>win7-x64;linux-x64;osx-x64</RuntimeIdentifiers>
```

Comment:
The removal of this line is crucial to allow cross-architecture publishing (i.e. publish for arm64 on x64 machine and vice versa). The issue is filed as https://github.com/dotnet/roslyn/issues/60744 maybe we'll get to the bottom of this but anyway, it seems it works without this line.

Reply:
I guess your research verified this - as far as I can understand this is a must so let's try it.

Reply:
Yes, it fails without removing this line. It most likely shouldn't as mentioned [here](https://github.com/dotnet/roslyn/issues/60744#issuecomment-1101731020) but we don't know for sure.